guides/anchor/4-compliance-server.md

```diff
@@ -0,0 +1,439 @@
+          }
+          else {
+            response.status(403).end();
+          }
+        });
+    });
+    ```
+
+    ```java
+    @POST
+    @Path("compliance/ask_user")
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response askUser(@FormParam("sender") String sender) {
+      JsonReader jsonReader = Json.createReader(new StringReader(sender));
+      JsonObject senderData = jsonReader.readObject();
+      jsonReader.close();
+
+      // You can do any checks that make sense here. For example, you may not
+      // want to share information with someone who has sanctions as above:
+      Permission permission = sanctionsDatabase.isAllowed(
+        senderData.getString("name"),
+        senderData.getString("address"),
+        senderData.getString("date_of_birth"));
+
+      if (permission.equals(Permission.Allowed)) {
+        return Response.ok().build();
+      }
+      else if (permission.equals(Permission.Denied)) {
+        return Response.status(403).build();
+      }
+      else {
+        // If you need to wait and perform manual checks, you'll have to create
+        // a way to do that as well.
+        notifyHumanForManualInformationSharing(senderData);
+        // The value for `pending` is a time to check back again in seconds.
+        return Response.accepted(
+          Json.createObjectBuilder()
+            .add("pending", 3600)
+            .build())
+          .build();
+      }
+    }
+    ```
+
+    </code-example>
+
+To keep things simple, we’ll add all three callbacks to the same server we are using for the bridge server callbacks. However, you can implement them on any service that makes sense in your infrastructure. Just make sure they’re reachable at the URLs in your config file.
+
+
+## Update Stellar.toml
+
+When other organizations need to contact your compliance server to authorize a payment to one of your customers, they consult your domain’s `stellar.toml` file for the address, just as when finding your federation server.
+
+For compliance operations, you’ll need to list two new properties in your `stellar.toml`:
+
+<code-example name="stellar.toml">
+
+```toml
+FEDERATION_SERVER = "https://www.your_org.com:8002/federation"
+AUTH_SERVER = "https://www.your_org.com:8003"
+SIGNING_KEY = "GAIGZHHWK3REZQPLQX5DNUN4A32CSEONTU6CMDBO7GDWLPSXZDSYA4BU"
+```
+
+</code-example>
+
+`AUTH_SERVER` is the address for the *external* port of your compliance server. Like your federation server, this can be any URL you like, but **it must support HTTPS and use a valid SSL certificate.**[^ssl]
+
+`SIGNING_KEY` is the public key that matches the secret seed specified for `signing_seed` in your compliance server’s configuration. Other organizations will use it to safely encrypt sensitive compliance data (like customer names, birthdates, and addresses) so that only you can read it and to verify that messages were actually sent by you.
+
```

Comment:
`SIGNING_KEY` is used to sign [Auth Request](https://www.stellar.org/developers/guides/compliance-protocol.html#auth-server) (`sig` param) so receiving organization can check whether it was really sent by the sender organization.
There will be also `ENCRYPTION_KEY` in stellar.toml file but we haven't decided on the format yet.